### PR TITLE
Registration database

### PR DESCRIPTION
**Pull request by madhulika9293**
Sir, please review my code and let me know any changes required.

### CODE REVIEW

File: WebP/project1/application.py [hunk 1]
```diff
@@ -42,22 +42,28 @@ def index():
...
 			db.session.commit()
-			return render_template("index.html",name=name) # index page with username if successful, for now
+			return render_template("index.html",name=fname) # index page with username if successful, for now
 	return render_template("registration.html")
 
+@app.route("/admin",methods=["GET"])
+def admin():
+	allusers = User.query.all()
+	for user in allusers:
+		print(user.fname+" "+user.password+" "+str(user.timestamp))
```
Comment (praveengarimella) [inline WebP/project1/application.py]: Pls avoid debugging statements and use logging.

File: WebP/project1/templates/admin.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<html>
+  <head>
+    <title>Admins Only Page</title>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+      crossorigin="anonymous"
+    />
+  </head>
```
Comment (praveengarimella) [inline WebP/project1/templates/admin.html]: generalize this into a template and import it?

File: WebP/project1/application.py [hunk 3]
```diff
@@ -42,22 +42,28 @@ def index():
...
 		elif not email:
 			return render_template("error.html",message="Please provide email")
 		elif not password:
 			return render_template("error.html",message="Please provide password")
 		else:
 			dt = datetime.datetime.now()
-			user = User(fname=name,email=email,password=password,timestamp=dt)
+			user = User(fname=fname,email=email,password=password,timestamp=dt)
 			db.session.add(user)
 			db.session.commit()
```
Comment (praveengarimella) [inline WebP/project1/application.py]: what if the user already exists?

**Review by praveengarimella — CHANGES_REQUESTED** [review]
Remove the non source files from the repo.

Fix all the issues that I've commented and send it back

File: WebP/project1/application.py (hunk 1 above)
Comment (madhulika9293) [reply WebP/project1/application.py]: I used logging module in python, sir

File: WebP/project1/templates/admin.html (hunk 2 above)
Comment (madhulika9293) [reply WebP/project1/templates/admin.html]: Yes, I've added layout.html and changed all other templates also accordingly.

File: WebP/project1/application.py (hunk 3 above)
Comment (madhulika9293) [reply WebP/project1/application.py]: Added a check and displayed it in the error message.

**Comment by praveengarimella** [comment]
Good job 👍

**Pull request merged**